crates/bevy_core/Cargo.toml

```diff
@@ -8,22 +8,24 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+default = ["bevy_math"]
+serialize = ["dep:serde"]
 
 [dependencies]
 # bevy
 bevy_app = { path = "../bevy_app", version = "0.9.0", features = ["bevy_reflect"] }
 bevy_ecs = { path = "../bevy_ecs", version = "0.9.0", features = ["bevy_reflect"] }
-bevy_math = { path = "../bevy_math", version = "0.9.0" }
 bevy_reflect = { path = "../bevy_reflect", version = "0.9.0", features = ["bevy"] }
 bevy_tasks = { path = "../bevy_tasks", version = "0.9.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.9.0" }
 
+# optional bevy dependencies
+bevy_math = { path = "../bevy_math", optional = true, version = "0.9.0" }
```

Comment:
bevy_math is only included in this crate because of type registration which depends on reflection, I'll add a comment to this effect and tweak with `#[cfg(all(feature = "bevy_math", feature = "bevy_reflect"))]` May as well be explicit about it since having bevy_math included here at all is dubious IMO.

Reply:
I agree with your skepticism here. Can you open an issue for this?

Reply:
Will do.